src/index.ts

```diff
@@ -1,6 +1,6 @@
 const invalidProtocolRegex = /^([^\w]*)(javascript|data|vbscript)/im;
 const htmlEntitiesRegex = /&#(\w+)(^\w|;)?/g;
-const htmlTabEntityRegex = /&tab;/gi;
+const htmlSpaceEntityRegex = /&(tab|newline);/gi;
```

Comment:
are there any other html entities that represent blank spaces?

I'm also wondering if we can just trash html entities entirely, or if there's a world in which an html entity is a valid part of a url?

I guess it _technically_ is for query params. Should we be using the `URL` class here and only remove the html entities from the domain itself, and leave them in the query params? (would depend again on whether or not we continue to support IE11)

Reply:
Oh, perfect :heart: